frontend/src/app/offers/offer-detail/offer-detail.component.css

```diff
@@ -0,0 +1,35 @@
+.jumbotron{
+    background-image: url({{offer.image}});
+}
+.card{
+    margin: 15px;
+    box-shadow: 5px 5px 5px #f0f4f8;
+    border-radius: 5px;
+}
+
+.btn{
```

Comment:
is bootstrap button not sufficient?

Reply:
I just realised I can use btn-lg, so I will delete this. By the way shouldn't they be a bit larger?
`<button type="button" class="btn btn-primary">Primary</button>`
`<button type="button" class="btn btn-primary btn-lg">Primary</button>`
![screenshot from 2017-11-27 10-04-03](https://user-images.githubusercontent.com/32519132/33258571-5947eaa8-d35a-11e7-873d-abc9e2290179.png)

Reply:
I think you can just leave btn. They probably should be bigger, indeed, but their size is defined in rem, so they "inherit" the size based on the font size of the root element (rem property). You can read about CSS rem here: https://www.sitepoint.com/understanding-and-using-rem-units-in-css/.

I think the root font-size should simply be increased.

Reply:
Ok, thank you. :)